include/tenstorrent/tt_arc_msg_type.h

```diff
@@ -21,8 +21,8 @@ extern "C" {
  * @{
  */
 
-/** @brief Enumeration listing the available host requests IDs the SMC can process*/
-enum msg_type {
+/** @brief Enumeration listing the available host request IDs the SMC can process*/
+enum tt_arc_msg_type {
```

Comment:
Maybe even `enum tt_arc_msg_e`. That's just splitting hairs though, so just a suggestion. The important thing is that the enum name is namespaced.

```suggestion
enum tt_arc_msg_type {
```

I guess technically, enum values could also be namespaced `TT_ARC_MSG_xxx`.